mapie/classification.py

```diff
@@ -500,6 +523,138 @@ def _add_random_tie_breaking(
         )
         return prediction_sets
 
+    def _fix_number_of_classes(
+        self,
+        est_classes: ArrayLike,
+        y_proba: ArrayLike
+    ) -> ArrayLike:
+        """
+        Fix shape of y_proba of validation set if number of classes
+        of the training set used for cross-validation is different than
+        number of classes of the original dataset y.
+
+        Parameters
+        ----------
+        est_classes : ArrayLike
+            Classes of the training set.
```

Comment:
Maybe `n_classes_training` is a more explicit name ?

Reply:
Modified.